test/libyul/yulInterpreterTests/external_call.yul

```diff
@@ -8,3 +8,5 @@
 // Memory dump:
 // Storage dump:
 //   0000000000000000000000000000000000000000000000000000000000000064: 0000000000000000000000000000000000000000000000000000000000000001
+// Calldata dump:
+//      0: 0000000000000000000000000000000000000000000000000000000000000000
```

Comment:
Just scrolled through this. Why is the call data here `0000000000000000000000000000000000000000000000000000000000000000`? Shouldn't it reflect `gas(), 0x45, 0x5, 0, 0x20, 0x30, 0x20`?

Reply:
ah ok, just read about the semantic of call. so you are providing 0x20 byte from address 0, right? and this is just zero, because there are just zeroes, right?

Reply:
Wouldn't be a very similar test nice, with actually providing call data?